### ISSUE TITLE
urn:cts:greekLit:tlg5026.msA.GKolov:6.112 

### ISSUE THREAD

**Issue by GKolov** [homermultitext/scholia-english#241]
Lemma (leave blank if no lemma):δᾶερ ἔπει μάλιστα

Comment: The [term] δᾶερ (''brother'') as  ἆνερ' ('man''.) Such words ending in -ερ are enclitic. They accent with an acute the pronoun σέ ('' you'')  , that means that they pronounce it with the unmodified accent, when  the pronoun is in relation to something. It is therefore true that the pronoun  here is distinctive. However, the common reading employed always  [this  construction] as enclitic, since a conjunction is not added '' ἔπει σ' εἴασεν Ἀχιλλεύς'' ('' since  Achilleus  did it to you''),  ''ἐπεί σε πρῶτα κιχάνω''  ('' since it's you that, first, I meet''), '' ἐπεί σε φυγῶν ᾿ϊκέτευσα'' ('' [  welcome in your ship] the exile, since I implored you''), ''ἐπεί σε λεόντα'' ( ''because [ Zeus made you] a lion ''),  this way also : '' ἐπεί σε μάλιστα· πόνος φρένας '' (''since it is you especially that  the  tiredness [has invaded] in your mind '').  And those who read it  this way seem to follow the first person, very probably; because of the vowel-sound the first person  is  demonstrated, the term pronounced with the unmodified accent and the enclitic.· if, however, the pronoun ἐμὲ (''me'' )in the accucative case, while keeping the -ε ,has the unmodified accent· but if it rejects it, it's enclitic. The [pronoun] which rejects the -ε was found, therefore, with the conjunction ἐπεί in the poet ,according to this construction: '' Ἕκτορ ἐπει με κατ αῖσαν''  ( '' Hector since, according to the law of destiny, you take me apart''), ''ἐπει μ' ἀφελεσθέ γε δόντες''  (''since you gave it to me, you take it back from me ''). It would be possible that we read it by pronouncing it as  an enclitic, if we agreed to this  construction: ΄΄ἐπεί σε μάλιστα΄΄ ('' since it is you especially '' ).

**Comment by GKolov** [comment]
The [term] δᾶερ (''brother'') like ἆνερ' ('man''.) Such words ending in -ερ are enclitic. They accent with an acute the pronoun σέ ('' you'') , that means that they pronounce it with the unmodified accent, when the pronoun is in relation to something. It is therefore true that the pronoun here is distinctive. However, the common reading employed always [this construction] as enclitic, since a conjunction is not added '' ἔπει σ' εἴασεν Ἀχιλλεύς'' ('' since Achilleus did it to you''), ''ἐπεί σε πρῶτα κιχάνω'' ('' since it's you that, first, I meet''), '' ἐπεί σε φυγῶν ᾿ϊκέτευσα'' ('' [ welcome in your ship] the exile, since I implored you''), ''ἐπεί σε λεόντα'' ( ''because [ Zeus made you] a lion ''), this way also : '' ἐπεί σε μάλιστα· πόνος φρένας '' (''since it is you especially that the tiredness [has invaded] in your mind ''). And those who read it this way seem to follow the first person, very probably; because of the vowel-sound the first person is demonstrated, the term pronounced with the unmodified accent and the enclitic; if, however, the pronoun ἐμὲ (''me'' )in the accucative case, while keeping the -ε ,has the unmodified accent, but if it rejects it, it's enclitic. The [pronoun] which rejects the -ε was found, therefore, with the conjunction ἐπεί in the poet ,according to this construction: '' Ἕκτορ ἐπει με κατ αῖσαν'' ( '' Hector since, according to the law of destiny, you take me apart''), ''ἐπει μ' ἀφελεσθέ γε δόντες'' (''since you gave it to me, you take it back from me ''). It would be possible that we read it by pronouncing it as an enclitic, if we agreed to this construction: ΄΄ἐπεί σε μάλιστα΄΄ ('' since it is you especially '' ).

**Comment by caseydue** [comment]
[Our transcription of this scholion leaves out a chunk of the text - not sure what happened there. @Eumaeus and @neelsmith, we need to fix the source text for this one! I have sent this one to Ineke as well because it is so technical and there are textual problems and it is just really hard! Here is what I have so far:]

Lemma: δᾶερ ἔπει μάλιστα

Comment: The [word] δᾶερ ("brother-in-law") [is] like ἄνερ ("husband"). Such words ending in -ερ are enclitic. [Note: Erbse emends ἐγκλιτικαι to κλητικαί "vocative."] They accent with an acute the pronoun σέ ("you"), that is, they pronounce it with an unmodified accent, when the pronoun is in relation to something. It is therefore true that the pronoun here is distinctive [[is this right for ἀντιδιασταλτική ??]]. The common reading, however, always reads [this construction] enclitically. What I mean is this. The [phrase] ἐπεί σε ...

τὸ ἐπεί σε εὑρέθη συνεχῶς οὕτως ἀνεγνωσμένον, ἐγκλιτικῶς ἀεί  [[I'm lost here!! Something like: "The [phrase] ἐπεί σε was found being read continuously in this way, always enclitically"]]

not with a conjunction added [as in the following examples]: ἔπει σ' εἴασεν Ἀχιλλεύς ("since Achilles allowed you"; Iliad 24.684), ἐπεί σε πρῶτα κιχάνω ("since it's you first that I meet"; Odyssey 13.228), ἐπεί σε φυγῶν ᾿ϊκέτευσα ("since in my flight I approached you as a suppliant"; Odyssey 15.277), ἐπεί σε λεόντα ("since [ Zeus made you] a lion"; Iliad 21.483). In this way also [here in Iliad 6.355]: ἐπεί σε μάλιστα πόνος φρένας (''since it is you especially that toil [has overwhelmed] in your mind"). And those who read it this way seem to follow the first person, very plausibly; for it is through the vowel-sound [??] that the first person is demonstrated, both the term pronounced with an unmodified accent and the enclitic [[Do I have this right??]]; if [[how to translate γε here?]] the pronoun ἐμὲ ("me") in the accusative case, while keeping the ε, has the unmodified accent, but if it casts it off, it's enclitic. The [pronoun] that casts off the ε was found, therefore, with the conjunction ἐπεί in the poet, in accordance with this construction: Ἕκτορ ἐπει με κατ αῖσαν'' ("Hector since you have reproached me in due measure"; Iliad 3.59), ἐπει μ' ἀφελεσθέ γε δόντες (''since you who gave her to me take her back from me"; Iliad 1.299). It would be plausible, following this logic, for us to read it enclitically: ἐπεί σε μάλιστα ("since it is you especially").

**Comment by GKolov** [comment]
Yes, Casey! It would be useful to make a list with the difficult passages which demand more corrections!
Just a remark! If I am not mistaken, the last phrase is missing  in the text.

**Comment by caseydue** [comment]
Yes! I overlooked the last phrase in the Venetus A. I fixed that part, at least, on my translation, and now I am just waiting to hear back from Ineke about some of the other places where I am lost. :)

**Comment by caseydue** [comment]
I received a lot of notes from Ineke on this one that I would like to preserve here. Then I will post a corrected translation following her guidance. 

NOTE 1: Erbse’s emendation is absolutely correct. The mistake is easily understood by the similarity in sound (because of iotacism) between -κλιτικ- and κλητικ- and because the rest of the note is about the difference in accentuation between orthotone and enclitic forms.

NOTE 2: ὀρθοτονέω: I know LSJ suggests the translation ‘with unmodified accent’, but I prefer ‘ with straight accentuation’. It is either used for ‘putting an acute accent on’ a form, or to distinguish forms with their normal / natural (‘ straight’) accent from enclitic ones. Vicente Bécares Botas, who wrote a useful (but not entirely reliable) ‘ Diccionario de terminología grammatical griega’ translates into Latin as ‘ recto accentu signare’, and cites the passages from Apollonius Dyscolus where ὀρθοτονέω is opposed to ἐγκλίνεσθαι (as in our passage), e.g. pron. 35.6 τῶν ἀντωνυμιῶν ἃς μὲν ὀρθοτονεῖσθαι, τουτέστι τὸν κατὰ φύσιν τόνον ἔχειν, ἃς δὲ ἐγκλίνεσθαι.

NOTE 3: πρός τι: The question in this scholion is what the correct accentuation of σε would be in this particular passage. The scholiast opposes the acute / straight accent (σέ) and defends, on the basis of other passages in Homer, and in particular of a clever argument about ἐμέ / με (where you can simply SEE the difference because ἐμέ has the additional ε) the enclitic reading here: ἐπεί σε.

The people who wish to keep the acute on σέ argue that σέ is here used ‘ relationally’, which the scholiast paraphrases right afterwards as ἀντιδιασταλτική: this means we’re having to do here with the emphatic form, which opposes  or contrasts ‘you’  to everyone else (σε μάλιστα). So, your translation is fine, just note that πρός τι is itself technical language.

NOTE 4 ‘therefore’: maybe not the best pick for μὲν οὖν. The structure of the argument is this: the position of the adversaries was complete after the report of their argument for orthotonesis ( ἐπεὶ πρός τί ἐστιν). Now the counter-argument of the scholiast begins (the οὖν part means he’s getting to the point). Μέν is opposed to μέντοι a bit further on. The scholiast first concedes a point in this μὲν οὖν phrase, and then explains why nonetheless and contrary to expectations (μέντοι) the word should still be enclitic. So I would translate: Now, (on the one hand) it is true that the pronoun is here used contrastively. However (mentoi) … 

NOTE 5 ἀντιδιασταλτικός for pronouns means ‘ contrastive’.

NOTE 6: why is [this construction] between brackets? ‘such a construction’. Note that construction here simply refers to a combination of words (epei se), not a ‘construction’  in our technical syntactic sense.

NOTE 7 actually, you got it right! He is now offering proof of his claim that the ‘common reading’ (in manuscripts/by scholars) of the Iliad text is ἐπεί σε wherever that phrase occurs. 

NOTE 8: There is one condition under which σε will have a straight accent, and that is when a σύνδεσμος (particle, not conjunction) follows. So: “ always enclitically, unless a particle follows” (reference is, of course, to enclitic particles).

NOTE 9: the analogy applied by people who read ἐπεί σε with enclitic σε is the use of the first person pronoun, where you can simply see the difference between enclitic με and orthotone ἐμέ. Με never has the acute (unless, once again, it is followed by another enclitic). So after ‘ very plausibly’, translate something like: ‘for the first person shows in its form (φωνή) whether it has an acute accent or is enclitic’ 

NOTE 10: εἴ γε· this combination in later Greek virtually always has a causal meaning: ‘since’.
‘ since the accusative ἐμέ, [keep going, in n. 12 the whole translation) 

NOTE 11: not ‘while’ but ‘when’ (this is again a condition for the rule): when keeping the ε (do not put the dash in front of the ε: if you want to use one it should go after it, since it is the initial ε of ἐμέ that’s under discussion here).

NOTE 12: maybe not ‘ if’ but ‘ whenever’ (opt. follows) it casts it off:
So: ‘ since the accusative ἐμέ, when keeping the ε, has a straight accent, but whenever it casts it off, it’s enclitic’

NOTE 13 I would suggest rephrasing the sentence ‘ the [pronoun] that casts of the ε etc.’ (and in particularly the use of ‘ therefore’): 
after having explained why the use of the first-person pronouns makes for such a great testcase, the scholiast now demonstrates that in fact it is the enclitic form με that is found following ἐπεί (and then cites the passage), τοίνυν marks this essential point in the argumentation. ‘Well, the accusative is indeed found in the poet to cast off the ε in this combination with the conjunction ἐπεί: (follows example)

NOTE 14 note that in the Greek here, the crucial accent has been omitted! The text should read ἐπεί με (2x).

**Comment by caseydue** [comment]
[I have marked invalid so we can fix the errors in the source text here.]

Lemma: δᾶερ ἔπει μάλιστα

Comment: The [word] δᾶερ ("brother-in-law") [is] like ἄνερ ("husband"). Such words ending in -ερ are enclitic. [Note: Erbse emends ἐγκλιτικαι to κλητικαί "vocative," which is surely correct.] They accent with an acute the pronoun σέ ("you"), that is, they pronounce it with straight accentuation, when the pronoun is in relation to something. Now it is true that the pronoun is here used contrastively. The common reading, however, always reads this construction enclitically. What I mean is this. The [phrase] ἐπεί σε was found being read continuously in this way, always enclitically, unless a particle follows [as in the following examples]: ἔπει σ' εἴασεν Ἀχιλλεύς ("since Achilles allowed you"; Iliad 24.684), ἐπεί σε πρῶτα κιχάνω ("since it's you first that I meet"; Odyssey 13.228), ἐπεί σε φυγῶν ᾿ϊκέτευσα ("since in my flight I approached you as a suppliant"; Odyssey 15.277), ἐπεί σε λεόντα ("since [ Zeus made you] a lion"; Iliad 21.483). In this way also [here in Iliad 6.355]: ἐπεί σε μάλιστα πόνος φρένας (''since it is you especially that toil [has overwhelmed] in your mind"). And those who read it this way seem to follow the first person, very plausibly; for the first person shows in its form whether it has an acute accent or is enclitic; since the pronoun ἐμὲ ("me") in the accusative case, when keeping the ε, has the straight accent, but whenever it casts it off, it's enclitic. Well, the accusative is indeed found in the poet to cast of the ε in this combination with the conjunction ἐπεί: Ἕκτορ ἐπεί με κατ αῖσαν'' ("Hector since you have reproached me in due measure"; Iliad 3.59), ἐπεί μ' ἀφελεσθέ γε δόντες (''since you who gave her to me take her back from me"; Iliad 1.299). It would be plausible, following this logic, for us to read it enclitically: ἐπεί σε μάλιστα ("since it is you especially").
